coral-component-masonry/src/scripts/Masonry.js

```diff
@@ -226,12 +226,18 @@ const Masonry = Decorator(class extends BaseComponent(HTMLElement) {
     this._selectionMode = validate.enumeration(selectionMode)(value) && value || selectionMode.NONE;
     this._reflectAttribute('selectionmode', this._selectionMode);
 
+    const isGrid = this.ariaGrid === ariaGrid.ON && this.parentElement;
+
     if (this._selectionMode === selectionMode.NONE) {
       this.classList.remove('is-selectable');
-      this.removeAttribute('aria-multiselectable');
+      if (isGrid) {
+        this.parentElement.removeAttribute('aria-multiselectable');
```

Comment:
Hi @majornista ,
coral-masonry is not wrapped in any parent element. There can be case where there is no parent element. check [this](https://opensource.adobe.com/coral-spectrum/playground/#?livereload=true&screen=vertical&code=MTY1LDE0NSwxNzcsNzgsMTk2LDQ4LDEyLDEzNCwyMzEsMTg3LDE2Nyw4LDIxNywyMTksMjM2LDQwLDIzNywyLDcyLDEwOCw0OCwzMiwzMywxOTgsOTIsMjI2LDE4NywyMzAsMTEyLDIzNCwyMDAsMTE4LDM3LDI1MCwyNDYsMjQ0LDkwLDIyMSwwLDIyNiwzOCwzOCw5MSwyNDYsMjM5LDIwNywyNTQsMTAxLDEyNywyNDcsMjQ4LDI0MiwyNDAsMjQ2LDI0MSwyNTAsMTAwLDYsNDUsMjE2LDIzOSwyNTMsNTMsNjQsNzIsMjUzLDIyMiwyNCwxNDMsMTIxLDI1Miw1MiwxMiwyMTYsODksMjA5LDI1LDY1LDYsMCwxODEsMTAyLDk2LDU2LDExOCwxMTgsODAsMTczLDExNCwyMzksMjgsODUsMjQsMTMzLDM4LDE0MiwyMDgsMTM0LDY4LDcsMTA0LDM1LDIxLDIzLDEzNywzLDU0LDgyLDMzLDQyLDc5LDE5Nyw4NSwxMiwyNDMsMTM3LDEwNSwyNiwxNDcsNzUsODksMjEyLDY5LDE0NSw3NywyMTAsNDYsMTUzLDExNywyMzUsNDYsMTM3LDE1NiwxNzEsMjYsMjI1LDI0OCw2MywyNDYsMjQ5LDEzOCw2MiwxMzksMjM3LDE4OSwyMTksMTc2LDEzOSw0MSwxODMsMTg1LDI0Miw3LDc0LDE3OSwxMzcsMjQsNjgsNTgsMTg3LDE2MSwyNiwyMDQsMTY3LDY1LDY1LDIxMiwxNzQsMTM1LDEwOCwxOTcsMTgsMTMyLDcwLDE1OCwyMDUsNjYsMTY3LDczLDU5LDEyMywyMDQsOTUsMTQ0LDE1NCw4LDE2MywyLDY3LDkwLDE0OCwxODcsMjIxLDc5LDEwMSwxNDcsMjEsNzQsMjU1LDEyLDEzNiwyMjgsMjIxLDMxLDE1NywyNywzLDIzOSwxOTYsMTUyLDExMCwxMiwyNTIsNDIsOTUsNzYsOTIsMTc0LDk1LDIwNSwxNzIsMTU5LDI1MCw2&)

If a parentElement is required, you can wrap it with div element. let me know if there is any concern.
Thank you

Reply:
Hi @Pareesh ,
is there any place in AEM cloud-ready instance where we can find that situation?
When the coral-masonry is not wrapped in any parent element?

Reply:
@Pareesh `isGrid` is defined at https://github.com/adobe/coral-spectrum/pull/250/files/11f064ab58dd4fe43fbcc171f28a7a09658377d7#diff-256c94a486661c28d9b00a5bd8e5420a737e2a89532647a425ba4fbe3aa4200eR229.  We only add `aria-multiselectable` to the `parentElement` when `this.ariaGrid === ariaGrid.ON` and there is a `parentElement`. With `this.ariaGrid === 'on'` the `coral-masonry` is already adding `role="grid"` to the `parentElement`. The reason for setting `role="grid"` on a `parentElement` rather than on the `coral-masonry` element itself, is to avoid having to change the structure of items within the masonry in order to properly implement the WAI-ARIA grid design pattern hierarchy, which could potentially introduce a breaking change for customers that override styles for masonry items.